src/Library/demos/Status Page/main.js

```diff
@@ -0,0 +1,19 @@
+import Gtk from "gi://Gtk?version=4.0";
+
+Gtk.init();
+
+const content = workbench.builder.get_object("content");
+const entry = workbench.builder.get_object("entry_1");
+const button = workbench.builder.get_object("submit_button");
+
+button.connect("clicked", () => {
+  const text = entry.get_text();
+  if (text === "Error") {
+    content.title = "Error 404";
+  } else if (text === "No Results") {
+    content.title = content.title;
+  } else {
+    content.title = "No content available for this keyword.";
+  }
+});
```

Comment:
This seems irrelevant to the current demonstration.

Reply:
This is a good idea to make the demonstration interactive, but I think there should be some more variation. Such as "Type to start searching", "Error connecting to server", "No results"...

Adding the keywords to the `placeholder-text` property of the entry will let developers know what terms to put in the entry, then they can see the status page change.